### PR TITLE
Create a custom point

### PR DESCRIPTION
**Pull request by patrickchoo1**

### CODE REVIEW

File: GUI/main_window.py [hunk 1]
```diff
@@ -42,11 +42,11 @@ def __init__(self, *args, **kwargs):
         self.Stack = QStackedWidget(self)
 
         body = Body(width, height)
-        visualizer = Visualizer()
+        #visualizer = Visualizer()
```
Comment (roninsharma25) [inline GUI/main_window.py:45]: You can remove lines of code that are commented out

File: GUI/widgets/plot_buttons.py [hunk 2]
```diff
@@ -36,3 +44,11 @@ def setPlotResetFlag(self, flag):
     
     def getPlotResetFlag(self):
         return self.plot_reset
+
+    def createPoint(self):  
+        x, done1 = QInputDialog.getInt(self, 'Input Dialog', 'Enter x-cooridinate')
```
Comment (roninsharma25) [inline GUI/widgets/plot_buttons.py:49]: Typo

**Review by roninsharma25 — CHANGES_REQUESTED** [review]
Can you modify the styles so that the text is more visible in the popup dialog? Could you also change the data point to be an x instead of a circle?

File: GUI/main_window.py (hunk 1 above)
Comment (ryanwmao) [reply GUI/main_window.py:45]: I don't think this should be commented out, it's the pod visualizer page

Comment (KelvinLWang) [reply GUI/main_window.py:45]: I agree with Ryan, the visualizer line shouldn't be commented out or removed

Comment (roninsharma25) [reply GUI/main_window.py:45]: Ryan and Kelvin are correct, my mistake. You should uncomment it when committing

**Pull request closed without merging**